src/QldbDriver.ts

```diff
@@ -156,36 +169,95 @@ export class QldbDriver {
      * @throws {@linkcode InvalidSessionException} When a session expires either due to a long running transaction or session being idle for long time.
      * @throws {@linkcode BadRequestException} When Amazon QLDB is not able to execute a query or transaction.
      */
-    async executeLambda(
-        transactionLambda: (transactionExecutor: TransactionExecutor) => any,
+    async executeLambda<Type>(
```

Comment:
Rather make this a `Promise<T>`. Using `Type` like this is confusing.

Reply:
Done.